modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4Utils.java

```diff
@@ -55,6 +57,19 @@ public static void setup() {
 
     }
 
+    private static AtomicBoolean isAvailableProcessorsSet = new AtomicBoolean();
+
+    /**
+     * Set the number of available processors that Netty uses for sizing various resources (e.g., thread pools).
```

Comment:
Sorry, I was skimming and didn't notice the HTTP server transport. I had presumed that that was netty3 which isn't even a thing in master anymore.

I agree it isn't worth logging anything since we invoke it twice.

Maybe we can check that the value matches if we don't invoke though? I just don't want to set the setting and not get any feedback if it doesn't take.